arch/M68K/M68KDisassembler.h

```diff
@@ -18,6 +18,8 @@ typedef struct m68k_info {
 	unsigned int type;
 	unsigned int address_mask; /* Address mask to simulate address lines */
 	cs_m68k extension;
+	uint8_t groups[8];
+	uint8_t groups_count;
```

Comment:
self noted: this requires change on Python bindings. if you can fix Python binding, that will be awesome.